examples/amqp_ssl_connect.c

```diff
@@ -95,7 +95,6 @@ int main(int argc, char const *const *argv) {
   die_on_amqp_error(amqp_connection_close(conn, AMQP_REPLY_SUCCESS),
                     "Closing connection");
   die_on_error(amqp_destroy_connection(conn), "Ending connection");
-  die_on_error(amqp_uninitialize_ssl_library(), "Uninitializing SSL library");
```

Comment:
Why is this change required?

Reply:
Deprecation notices complaining when building examples.

Reply:
Please send me a separate PR for this change.